### PR TITLE
Merge deployment doc changes to master

### PR DESCRIPTION
**Pull request by nilminiwso2**
Merging https://github.com/wso2/docs-apim/pull/3953 to master.

### CODE REVIEW

File: en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md [hunk 1]
```diff
@@ -1,359 +1,104 @@
...
     </tr>
     <tr>
         <td>
-            Key Manager
+            Gateway Profile
         </td>
         <td>
-            Responsible for all security and key-related operations.
+            <p>Only starts the components related to the API Gateway.</p>
+<p>Use this when the API Gateway acts as a worker node in a cluster. This profile starts the backend features for data processing and communicates with the management node.</p>
```
Comment (Mariangela) [inline en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:25]: @tharindu1st , please verify this description. Is it correct to use the terminology "worker node" here?

Comment (tharindu1st) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:25]: lets remove worker node concept from docs.

Comment (Mariangela) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md:25]: @nilminiwso2 , please note

Comment (nilminiwso2) [reply en/docs/install-and-setup/setup/distributed-deployment/understanding-the-distributed-deployment-of-wso2-api-m.md]: fixed

**Review by Mariangela — APPROVED** [review]

**Pull request merged**